Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM node:14-alpine
+FROM node:16-alpine
 
 ENV NODE_ENV production
 
 WORKDIR /app
 COPY package.json .
 COPY package-lock.json .
-RUN npm ci
+RUN node_modules/ node_modules
```

Comment:
Run node_modules?

Reply:
Jeg har pølsefingre. Byttet ut med "COPY" nå. :) 

Kort om hvorfor. Det virker som om installasjon og kjøring av husky ikke funker så bra i node:16-alpine. Siden vi ikke trenger å kjøre husky (eller andre post-install scripts) tenker jeg det er bedre (og raskere) å dra inn node_modules fra github actions. Er også dette vi gjør i de andre appene våre hvor vi har behov for å dra inn eks nextjs eller express i dockerimage.